docs/scripts/reportBrokenLinks.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-console */
 const path = require('path');
 const fse = require('fs-extra');
-const { createRender } = require('@mui/markdown');
+const { createRender } = require('docs/packages/markdown');
```

Comment:
Based on the context of https://github.com/mui/mui-x/pull/6497, doing this will resolve to https://github.com/mui/mui-x/blob/next/docs/packages/markdown/index.js which doesn't exist.